### ISSUE TITLE
Null check operator used on a null value (Version 0.3.1)

### ISSUE THREAD

**Issue by naamapps** [bnxm/material_floating_search_bar#30]
The package is throwing an error on versions 0.3.0 and 0.3.1
Can't use the page the search bar is on because of this error:
```
I/flutter ( 5040): Null check operator used on a null value
I/flutter ( 5040): #0      FloatingSearchBarTransition.lerpMaxWidth (package:material_floating_search_bar/src/floating_search_bar_transition.dart:52)
I/flutter ( 5040): #1      FloatingSearchBarState._buildSearchBar (package:material_floating_search_bar/src/floating_search_bar.dart:617)
I/flutter ( 5040): #2      FloatingSearchBarState.build.<anonymous closure> (package:material_floating_search_bar/src/floating_search_bar.dart:587)
```

**Comment by bnxm** [comment]
Should be fixed with 0.3.2

**Comment by naamapps** [comment]
@bnxm  
Yep works perfect now.
Thanks :)
